src/components/BiobankExplorerContainer.vue

```diff
@@ -42,13 +42,16 @@
     },
     computed: {
       /* A computed 'compound' object for watching all filters */
+      // TODO: Use an action to retrieve the collections when the filters are changed
```

Comment:
I think this is a separate issue, so I've kept things as is, but changing the filters should be an action that triggers this.